bundle-size/app.js

```diff
@@ -606,11 +608,47 @@ module.exports = app => {
         const errorMessage =
           `ERROR: Failed to create the bundle-size/${bundleSizeFile} file in ` +
           'the build artifacts repository on GitHub!\n' +
-          `Error message was: ${error.message}`;
+          `Error message was: ${error}`;
         app.log(errorMessage);
         return response.status(500).end(errorMessage);
       }
     }
+
+    const jsonBundleSizeFile = `${headSha}.json`;
+    try {
+      await getBuildArtifactsFile(userBasedGithub, jsonBundleSizeFile);
+      app.log(
+        `The file bundle-size/${jsonBundleSizeFile} already exists in the ` +
+          'build artifacts repository on GitHub. Skipping...'
+      );
+      return response.end();
+    } catch (unusedException) {
+      // The file was not found in the GitHub repository, so continue to
+      // create it...
+    }
+
+    try {
+      const jsonBundleSizeText = JSON.stringify({
+        'dist/v0.js': brotliBundleSize,
+      });
+      await storeBuildArtifactsFile(
+        userBasedGithub,
+        jsonBundleSizeFile,
+        jsonBundleSizeText
+      );
+      app.log(
+        'Stored the new bundle size JSON file in the artifacts repository on ' +
```

Comment:
Suggestion: Include the  filename in the log

Reply:
Good point. Done!